modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -133,17 +133,15 @@ class {{{classname}}} {
   	          {{/items.complexType}}
           {{/items.isArray}}
           {{^items.isArray}}
-        {{{name}}}: json[r'{{{baseName}}}'] == null
-          ? null
-          : {{{complexType}}}.mapFromJson(json[r'{{{baseName}}}']),
+        {{{name}}}: json[r'{{{baseName}}}'] as {{{datatype}}},
```

Comment:
Are we sure that this can only be a primitive type here?

Reply:
This is the map of maps case...

I don't see this case in the petstore, but my project had this case:
![image](https://user-images.githubusercontent.com/7496871/102776642-db9e1a00-436d-11eb-88a7-5e7727b8c93c.png)

I'm gonna change this line to the same as the list treatment (above)
```mustache
          {{#items.isMap}}
            {{{name}}}: json[r'{{{baseName}}}'] == null
              ? null
                {{#items.complexType}}
                    : {{items.complexType}}.mapFromJson(json[r'{{{baseName}}}']),
                {{/items.complexType}}
                {{^items.complexType}}
                    : (json[r'{{{baseName}}}'] as Map).cast<String, Map>(),
                {{/items.complexType}}
          {{/items.isMap}}
          {{^items.isMap}}
              {{{name}}}: json[r'{{{baseName}}}'] // primitive? Probably Map<String, primitive>
          {{/items.isMap}}
```

This is what it will generate:
![image](https://user-images.githubusercontent.com/7496871/102778165-d55d6d00-4370-11eb-9447-a5da40c3f8ca.png)

Its that ok?